runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -105,9 +105,12 @@
 
 	// Chest loot handling
 	private static final String CHEST_LOOTED_MESSAGE = "You find some treasure in the chest!";
+	private static final String LARRAN_LOOTED_PATTERN = "You have opened Larran's ([a-z]+) chest .*";
```

Comment:
Make this a Pattern (eg. `Pattern.compile("You have opened Larran's \\w+ chest")`) and match against that instead of using `String.matches()` below.

Reply:
Made those changes